panel/src/panel/content.js

```diff
@@ -38,6 +38,11 @@ export default (panel) => {
 		 * Removes all unpublished changes
 		 */
 		async discard() {
+			// avoid requests if there is no API defined
+			if (!this.api) {
+				return;
```

Comment:
`Questions:` What were the cases where this was happening?

`Suggestion:` Shouldn't this throw a console error at least as something is misconfigured then?

Reply:
The last input happened and the save call is scheduled but the user already navigated quickly enough to a new view without an API endpoint. I ran into that. 

Thinking about it while typing, I realize that there's even a worse option. If you switch fast enough to a different view with a different API endpoint, the call would go to that one instead. This means we really need to cancel the last call when moving away.

Reply:
But if we cancel it, the content gets lost. We rather need to ensure that the request succeeds to the right endpoint.

Reply:
I think the issue lies somewhere here: https://github.com/getkirby/kirby/blob/v5/changes/develop/panel/src/panel/content.js#L179-L189 

We watch the endpoint for a change to send a last save request if any new input came in after the last request. But since we call it when `content.api` changes, the call made then won't have the old API endpoint anymore. I think we would need to pass the old API endpoint string there somehow to the save method.

Reply:
Also seeing that the watcher fires even on the initial load which isn't great.